src/main/java/org/opensearch/testcontainers/OpenSearchContainer.java

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
Honestly my biggest concern here is that things will go very badly for people with a case-insensitive file system (which I think is the default on MacOS?) since the file and class names only differ in the case of a single letter.

Reply:
As expected, it messes things on Windows, I believe we should change the approach here :(